### ISSUE TITLE
Destroy Cumulus development deployment in account csdap-cumulus-uat-7469 for Jayanthi

### ISSUE THREAD

**Issue by chuckwondo** [NASA-IMPACT/csdap-cumulus#95]
Only once the Prod, UAT, and sandbox deployments in the new CBA accounts are fully operational, destroy Jayanthi's development deployment in account [csdap-cumulus-uat-7469](https://cloud.earthdata.nasa.gov/portal/project/171) by following the "Destroying a Deployment" instructions in this repo's `docs/OPERATING.md` file.

**Acceptance criterion:** all AWS resources related to Jayanthi's development deployment in account csdap-cumulus-uat-7469 are destroyed.

**Comment by krisstanton** [comment]
@jsrikish is currently blocked by this same issue I had.

https://github.com/NASA-IMPACT/csdap-cumulus/pull/119#issuecomment-1438902353

**Comment by chuckwondo** [comment]
Why is it looking for Kris's table? What are the contents of your `.env` file?

**Comment by jsrikish** [comment]
_Sorry.  I did a cut-n-paste from my convo with Kris & he sent his error msg_
**********************************************************************
Error: error reading Dynamodb Table (cumulus-jayanthi-CollectionsTable): ResourceNotFoundException: Requested resource not found: Table: cumulus-jayanthi-CollectionsTable not found
 on .terraform/modules/cumulus/tf-modules/archive/ingest-reporting.tf line 404, in data "aws_dynamodb_table" "collections"

**Comment by chuckwondo** [comment]
I manually deleted your tables, which should get past this issue. Try again.

**Comment by chuckwondo** [comment]
@jsrikish, now that you mentioned to me that `make nuke` finished successfully, make sure you follow the remaining steps for destroying your deployment as described in `docs/OPERATING.md`.

**Comment by jsrikish** [comment]
aws s3 rb s3://csdap-cumulus-jayanthi-tfstate  --force
remove_bucket failed: s3://csdap-cumulus-jayanthi-tfstate An error occurred (BucketNotEmpty) when calling the DeleteBucket operation: The bucket you tried to delete is not empty. You must delete all versions in the bucket

Tried:
aws s3 rm  s3://csdap-cumulus-jayanthi-tfstate/*
aws s3 rb s3://csdap-cumulus-jayanthi-tfstate  --force
remove_bucket failed: s3://csdap-cumulus-jayanthi-tfstate An error occurred (BucketNotEmpty) when calling the DeleteBucket operation: The bucket you tried to delete is not empty. You must delete all versions in the bucket

aws s3api list-object-versions --bucket  "csdap-cumulus-jayanthi-tfstate" --output=json --query='{Objects: Versions[].{Key:Key,VersionId:VersionId}}'
Lists several version IDs.  
I tried to delete objects & version IDs
aws s3api delete-objects --bucket csdap-cumulus-jayanthi-tfstate --delete "$(aws s3api list-object-versions --bucket "csdap-cumulus-jayanthi-tfstate" --output=json --query='{Objects: Versions[].{Key:Key,VersionId:VersionId}}')"

aws s3 rb s3://csdap-cumulus-jayanthi-tfstate  --force.   --Fails
